addons/website/views/website_templates.xml

```diff
@@ -65,6 +65,9 @@
         </t>
         <t t-set="x_icon" t-value="website.image_url(website, 'favicon')"/>
     </xpath>
+    <xpath expr="//head[last()]" position="after">
```

Comment:
Should it not be `<xpath expr="//head" position="inside">` ? Strange if it works between the head and the body...